bin/crew

```diff
@@ -1963,10 +1975,23 @@ end
 
 def build_command(args)
   abort 'Unable to locate local repo root directory. Change to a local chromebrew git repo directory and try again.'.lightred unless Dir.exist? CREW_LOCAL_REPO_ROOT
-  abort "Unable to locate local build directory #{CREW_LOCAL_BUILD_DIR}.".lightred unless Dir.exist? CREW_LOCAL_BUILD_DIR
+  unless Dir.exist? CREW_LOCAL_BUILD_DIR
+    if @opt_force
+      puts "Attempting to creating local build directory at #{CREW_LOCAL_BUILD_DIR} .".orange
+      FileUtils.mkdir_p CREW_LOCAL_BUILD_DIR
+    else
+      abort "Unable to locate local build directory #{CREW_LOCAL_BUILD_DIR}. It will be created if you build with the '-f' flag.".lightred
+    end
+  end
   abort "#{CREW_LOCAL_BUILD_DIR} is not writable.".lightred unless File.writable?(CREW_LOCAL_BUILD_DIR)
   args['<name>'].each do |name|
-    @pkgName = name
+    # If a package file is explicitly passed, then use that package file, whereever it is.
+    if name.include?('.rb') && File.file?(name)
```

Comment:
Does `name` ever include the `.rb` extension?

Reply:
Yes because name is args....